cmd-ref: plots: flexible plots docs (#3691)
iterative/dvc.org: content/docs/command-reference/plots/modify.md

# plots modify

> Modify display properties of
> [stage plots](/doc/command-reference/plots#stage-plots) files.

Reviewer: ```suggestion
Modify display properties of data-series
[plots](/doc/command-reference/plots) defined in <abbr>stages</abbr>.
```

# plots modify

Modify display properties of
[stage plots](/doc/command-reference/plots#stage-plots) files.

> ⚠️ Note that this command can modify only data-series plots. It has no effect

> > on image-type plots and standalone plots. See
> > [Types of plots](/doc/command-reference/plots#description).

Reviewer: Users can click previous links to find out about general info on plots, no need to update this link since that section is gone (we can remove it). Committing
```suggestion
> on image-type plots and standalone plots.
```

                        target

positional arguments:

>   target                Stage plot to set properties to.

Reviewer: ```suggestion
  target                Plot file to set properties for
```

Reviewer: ☝🏼  The description may need a similar update 👇🏼

Author: Done

                        target

positional arguments:
  target                Plots file to set properties for

>                         (defined at the stage level)

Reviewer: `plots modify target`
